.env.dist

```diff
@@ -34,4 +34,6 @@ CLOUDINARY_API_SECRET=A_IlqhvQ4DtmzkupP9rwCshavTo
 PORT=8000
 
 CARMD_PARTNER_TOKEN="103b504228694c7eba76bc9f0ffa0abb"
-CARMD_APIKEY="Basic NzNlZTI5NjAtZmVkOS00MTMxLWI0ZjctODc2ZTA4NjE3ODZm"
\ No newline at end of file
+CARMD_APIKEY="Basic NzNlZTI5NjAtZmVkOS00MTMxLWI0ZjctODc2ZTA4NjE3ODZm"
+
+SMTP_PASSWORD=ncTkaSqq5KLZH0Ddux7V4w
```

Comment:
Please remove this file,it should not be in version control as it exposes passwords and API keys